src/main/resources/assets/embeddiumplus/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+{
+  "embeddium.plus.options.plus": "附加",
+  "embeddium.plus.options.common.off": "关",
+  "embeddium.plus.options.common.simple": "简单",
+  "embeddium.plus.options.common.advanced": "高级",
+  "embeddium.plus.options.common.slow": "慢",
+  "embeddium.plus.options.common.fast": "快",
+  "embeddium.plus.options.common.faster": "更快",
+  "embeddium.plus.options.common.realtime": "实时",
+  "embeddium.plus.options.displayfps.title": "显示FPS",
+  "embeddium.plus.options.displayfps.desc": "显示当前FPS. 高级模式还显示最低FPS以及15秒的平均FPS，这对判断性能很有用。",
+  "embeddium.plus.options.displayfps.position.title": "FPS显示位置",
+  "embeddium.plus.options.displayfps.position.desc": "将FPS显示偏移几个像素",
+  "embeddium.plus.options.darkness.title": "真正的黑暗",
+  "embeddium.plus.options.darkness.desc": "使世界其他地方更加逼真地黑暗。不影响日间照明或火把照明。",
+  "embeddium.plus.options.darkness.mode.title": "真实黑暗方式",

```

为啥两者对 True Darkness 的翻译不一致？副词 `逼真地（realistically）` 形容形容词 `黑暗（dark）`，我感觉可以翻译成名词 `黑暗（dark）` 后跟副词 `逼真（realistical）` 的句式。
Why are the two translations of True Darkness inconsistent? The adverb `realistically` describes the adjective `dark`, I think it can be translated into a sentence pattern of the noun `dark` followed by the adverb `realistical`.
```suggestion
  "embeddium.plus.options.darkness.title": "真实黑暗",
  "embeddium.plus.options.darkness.desc": "使世界其他地方的黑暗更为逼真。这不会影响日间照明或火把照明。",
  "embeddium.plus.options.darkness.mode.title": "真实黑暗的模式",

```